components/applications-service/integration_test/services_by_sg_test.go

```diff
@@ -90,6 +90,7 @@ func TestGetServicesBySGSingleService(t *testing.T) {
 						Status:       applications.ServiceStatus_RUNNING,
 						HealthCheck:  applications.HealthStatus_OK,
 						Application:  a, Environment: e, Fqdn: "",
+						Channel: "stable", Site: "test",
```

Comment:
Should we make all of these channel and site variables global? (since they are all the same) Just like `a` and `e` for application and environment. (Not really the best variable names though haha 😂 )